tests/integration/conftest.py

```diff
@@ -571,6 +551,67 @@ async def app_with_allowed_plugins_fixture(
     await application.reset_config(to_default=["allowed-plugins"])
 
 
+@pytest.fixture(scope="module", name="ldap_settings")
+def ldap_settings_fixture() -> TestLDAPSettings:
+    """LDAP user for testing."""
+    return TestLDAPSettings(
+        container_port=1389,
+        username="customuser",
+        password=secrets.token_hex(16),
+    )
+
+
+@pytest_asyncio.fixture(scope="module", name="ldap_server")
+async def ldap_server_fixture(
+    model: Model, kube_apps_client: kubernetes.client.AppsV1Api, ldap_settings: TestLDAPSettings
+):
+    """Testing LDAP server pod."""
+    container = kubernetes.client.V1Container(
+        name="ldap",
+        image="bitnami/openldap:latest",
```

Comment:
I suggest you use a more restrictive tag here. A breaking change in image would be difficult to track

Reply:
Btw, there's an LDAP charm, IDK if you've considered using it https://charmhub.io/openldap-charmers-openldap

Reply:
Tagged with more restrictive image! As of the charm, I think the test case only requires a lightweight ldap server for testing, so it could be easier to deal with a kubernetes deployment than a fully deployed charm. Would that be alright?

Reply:
I guess